packages/tasks/src/tasks/image-to-image/data.ts

```diff
@@ -93,7 +93,7 @@ const taskData: TaskDataCustom = {
 		},
 	],
 	summary:
-		"Image-to-image is the task of transforming a source image to match the characteristics of a target image or a target image domain. Any image manipulation and enhancement is possible with image to image models.",
+		"Image-to-image is the task of transforming an input image through a variety of possible manipulations and enhancements, such as super resolution, image inpainting, colorization and more. Image-to-image pipelines can also be used in text-to-image tasks, to provide visual guidance to the text-guided generation process.",
```

Comment:
```suggestion
		"Image-to-image is the task of transforming an input image through a variety of possible manipulations and enhancements, such as super-resolution, image inpainting, colorization, and more. 
```

Can we move this with a bit more explanation to the top of about.md: Image-to-image pipelines can also be used in text-to-image tasks, to provide visual guidance to the text-guided generation process."